lib/core/src/Cardano/Wallet/Primitive/Types.hs

```diff
@@ -1283,11 +1283,14 @@ data SlotParameters = SlotParameters
 data SyncProgress
     = Ready
     | Syncing !(Quantity "percent" Percentage)
+    | Dead
     deriving (Generic, Eq, Show)
 
 instance NFData SyncProgress
 
 instance Ord SyncProgress where
+    Dead <= _ = True
+    _ <= Dead = False
```

Comment:
could you give rationale behind those ^^^ ?

Reply:
We needed an ordering ^^" ... So I thought we might as well just say that "Dead" is smaller than anything else. That's purely arbitrary.

Reply:
yeah, I know we need ordering, just wanted if there is some thinking behind this that will address some edge case, etc.

Reply:
Nope, purely arbitrary here :man_shrugging: